File: programmers/python/Lv0/코드처리하기.py
def solution(code):
    answer = ""
    mode = 0
    for i in range(len(code)):
        if code[i] == "1":
            mode = abs(mode-1)
        elif i % 2 == mode:
            answer += code[i]
    if answer == "":
        return "EMPTY"
    return answer

File: programmers/java/Lv0/코드처리하기.java
class Solution {
    public String solution(String code) {
        Boolean mode = false;
        String ret = "";
        
        for (int i=0; i<code.length(); i++) {
            if (code.charAt(i) == '1') {
                mode = !(mode);
            }
            else {
                if (mode && (i%2==1)) {
                    ret += code.charAt(i);
                }
                else if ((!mode) && (i%2==0)) {
                    ret += code.charAt(i);
                }
            }
        }
        if (ret.equals("")) {
            ret = "EMPTY";
        }
        return ret;
    }
}